sonaranalyzer-dotnet/rspec/cs/S1006_c#.json

```diff
@@ -7,6 +7,7 @@
     "constantCost": "2min"
   },
   "tags": [
+    "cppcoreguidelines",
```

Comment:
@loic-joly-sonarsource this sounds familiar - C++ tags should not appear on other rules, right? I think this was mentioned before